src/utils/hots-commons.ts

```diff
@@ -0,0 +1,23 @@
+export enum HeroRoles {
+  Assassin,
+  Warrior,
+  Specialist,
+  Support
+}
+
+export interface Skill {
+  name: string;
+  level: number;
+  description: string;
+}
+
+export interface Build {
+  name: string;
+  skills: Array<Skill>;
+}
+
+export interface Hero {
+  name: string;
+  role: HeroRoles|Array<HeroRoles>;
```

Comment:
Maybe just `role: Array<HeroRoles>;` to avoid later `if (typeof hero.role === 'Array') {} else {}`?